tests.py

```diff
@@ -311,6 +311,28 @@ def test_import_from_qtwidgets():
     assert QPushButton.__name__ == "QPushButton", QPushButton
 
 
+def test_pyside2uic():
```

Comment:
...since I was copying pyside2uic from Python 3 to Python 2 I figured it would be good to test that. Perhaps that should've been more clear in my test.